### ISSUE TITLE
Hungarian localizations not working

### ISSUE THREAD

**Issue by koltai75** [flutter-form-builder-ecosystem/form_builder_validators#4]
Hello! I am trying to use Hungarian localization, but it is not working. Under flutter_form_builder/lib/l10n/ there is an intl_hu.arb file containing Hungarian localization, but in my app English is displayed. In the same folder I don't see messages_hu.dart and messages_all.dart does not contain Hungarian language.

**Comment by CaLouro** [comment]
I'm having the same problem with the pt locale. The localizations _seems_ to initialize correctly but when a validator message is called it defaults to English.

Interestingly enough I've been using the same Localizely localization engine on my code and my custom localizations load just fine.

I'll keep an eye to see anything relevant to this.

**Comment by CaLouro** [comment]
Just a quick update. I've tested using the validator localizations on a empty project and it worked fine but it still won't work with a bigger project with custom localization.


Used versions and working code
```
flutter_form_builder: ^7.6.0
form_builder_validators: ^8.3.0
```

```dart
import 'package:flutter/material.dart';
import 'package:flutter_form_builder/flutter_form_builder.dart';
import 'package:flutter_localizations/flutter_localizations.dart';
import 'package:form_builder_validators/form_builder_validators.dart';

void main() {
  runApp(const Sandbox());
}

class Sandbox extends StatelessWidget {
  const Sandbox({super.key});

  @override
  Widget build(BuildContext context) {
    return const MaterialApp(
      title: 'Sandbox',
      supportedLocales: [
        Locale('en'),
        Locale('pt'),
      ],
      localizationsDelegates: [
        ...GlobalMaterialLocalizations.delegates,
        FormBuilderLocalizations.delegate,
      ],
      locale: Locale('pt'),
      home: SandBoxWidget(),
    );
  }
}

class SandBoxWidget extends StatefulWidget {
  const SandBoxWidget({Key? key}) : super(key: key);

  @override
  State<SandBoxWidget> createState() => _SandBoxWidgetState();
}

class _SandBoxWidgetState extends State<SandBoxWidget> {
  final key = GlobalKey<FormBuilderState>();

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: FormBuilder(
        key: key,
        child: ListView(
          children: [
            FormBuilderTextField(
              name: 'name',
              validator: FormBuilderValidators.required(),
            ),
            ElevatedButton(
              onPressed: () => key.currentState?.validate(),
              child: const Text('Test'),
            ),
          ],
        ),
      ),
    );
  }
}
```

**Comment by CaLouro** [comment]
Upon further debugging, it seems like the problem I'm having is due to a limitation on the Intl package which is quite unusual for such an important package.

TLDR is: CompositeMessageLookup from Intl will not add the FormBuilderValidators localization messages because my app already added a locale with my custom localizations messages to the lookup library, leaving this package without proper localization support.

I don't know if this was the problem with the author's code and if it's not I will open another issue to have this problem registered properly.

**Comment by koltai75** [comment]
The project in which I had this problem also contained some custom localization. I checked the code, I ended up with removing the custom localization from `localizationsDelegates`.

**Comment by CaLouro** [comment]
Did form_builder_validators' localization work after that?

**Comment by koltai75** [comment]
Sorry, I don't remember. It was almost a year ago and I did not keep this code.

**Comment by CaLouro** [comment]
No problem. Just wanted another confirmation that this was the issue.

**Comment by deandreamatias** [comment]
Maybe related with https://github.com/flutter-form-builder-ecosystem/form_builder_validators/issues/14 ?

**Comment by CaLouro** [comment]
It is actually, but their solution didn't work for me and downgraded a bunch of my dependencies.

**Comment by deandreamatias** [comment]
I think that this is solved with #22
